src/accordion.tsx

```diff
@@ -263,14 +275,13 @@ interface BoxedAccordionItemProps extends AccordionItemContentProps {
     isInverse?: boolean;
 }
 
-export const BoxedAccordionItem = React.forwardRef<HTMLDivElement, BoxedAccordionItemProps>(
+export const BoxedAccordionItem = React.forwardRef<TouchableElement, BoxedAccordionItemProps>(
     ({dataAttributes, isInverse, ...props}, ref) => (
         <Boxed
             isInverse={isInverse}
-            ref={ref}
```

Comment:
This was wrong. The ref should be passed to the content. Otherwise, calling ref.current.click() won't affect the Accordion

Reply:
Is this a real use case? I mean, it feels weird to me to control the open/close of the Accordion using a ref.

Reply:
I'm not sure. But even if there is no real use case, the ref logic was already implemented, so removing it would be a breaking change. I've just fixed it so that it actually works as expected

Reply:
Discussed offline. We will put the ref in the topmost element of the component. We believe it should be like this because refs are usually used for other things instead of clicking the element (like checking component's visibility)